_episodes/00-quickstart.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Quickstart Guide"
+teaching: 5
+exercises: 10
+questions:
+
+- "How do I activate the ESMValTool environment?"
+- "How do I configure the ESMValTool?"
+- "How do I run a recipe?"
+
+objectives:
+
+- "Activate the ESMValTool environment"
+- "Configure the ESMValTool"
+- "Successfully run a recipe"
+
+keypoints:
+- "See the installation guide for activating the ESMValtool environment."
```

Comment:
Why we point the user to the installation guide? The main idea in issue #245 is that esmvaltool is already installed.